tests/rest/transactioninfo_test.py

```diff
@@ -116,6 +116,10 @@ def test_transactioninfo_api(self):
             url='{0}/transactioninfo/multisearch?instrument_group_id={1}&project_id=1234a'.format(self.url, 1001))
         self.assertEqual(req.status_code, 200)
 
+        req = requests.get(
+            url='{0}/transactioninfo/summary/{1}/{2}'.format(self.url, '2015-07-01', '2019-07-30'))
```

Comment:
@kauberry I'm wondering if your testing isn't catching the state of the metadata, the `updated` fields get updated to the current time when the Travis CI pipeline gets run. So, maybe set your end date to a calculated date that always returns tomorrow?